<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>config::VariableComboDialog</class>
 <widget class="QDialog" name="config::VariableComboDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>534</width>
    <height>384</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Edit  Variable</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>1</x>
     <y>4</y>
     <width>484</width>
     <height>335</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="VariableLabel">
            <property name="text">
             <string>Variable Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="VariableText">
            <property name="toolTip">
             <string extracomment="The name of the Variable - use CAPS"/>
            </property>
            <property name="inputMethodHints">
             <set>Qt::ImhUppercaseOnly</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="UnitsLayout">
          <item>
           <widget class="QLabel" name="UnitsLabel_2">
            <property name="text">
             <string>Units</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="UnitsText"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="SRLayout">
          <item>
           <widget class="QLabel" name="SRLabel">
            <property name="text">
             <string>Sample Rate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="SRText"/>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>168</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QCheckBox" name="calFileCheckBox">
            <property name="text">
             <string>Use Calfile</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_7">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>17</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_5">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="CalLabel">
          <property name="text">
           <string>CalLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal1Layout">
          <item>
           <widget class="QLabel" name="Cal1Label">
            <property name="text">
             <string>C0</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib1Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal2Layout">
          <item>
           <widget class="QLabel" name="Cal2Label">
            <property name="text">
             <string>C1</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib2Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal3Layout">
          <item>
           <widget class="QLabel" name="Cal3Label">
            <property name="text">
             <string>C2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib3Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal4Layout">
          <item>
           <widget class="QLabel" name="Cal4Label">
            <property name="text">
             <string>C3</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib4Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal5Layout">
          <item>
           <widget class="QLabel" name="Cal5Label">
            <property name="text">
             <string>C4</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib5Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="Cal6Layout">
          <item>
           <widget class="QLabel" name="Cal6Label">
            <property name="text">
             <string>C5</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Calib6Text">
            <property name="text">
             <string>CalLabel With size  </string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_6">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_4">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="LongNameLayout">
      <item>
       <widget class="QLabel" name="LongNameLabel">
        <property name="text">
         <string>Long Name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LongNameText"/>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_bottom">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>168</width>
        <height>13</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QDialogButtonBox" name="buttonBox">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="standardButtons">
         <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>VariableText</tabstop>
  <tabstop>LongNameText</tabstop>
  <tabstop>SRText</tabstop>
  <tabstop>UnitsText</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>config::VariableComboDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>248</y>
    </hint>
    <hint type="destinationlabel">
     <x>370</x>
     <y>220</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>config::VariableComboDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>243</y>
    </hint>
    <hint type="destinationlabel">
     <x>12</x>
     <y>226</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>newSensor(QString)</slot>
  <slot>setDevice(int)</slot>
 </slots>
</ui>
